jwt-access-control/README.md

```diff
@@ -259,27 +252,19 @@ send it automatically to our API.
 
 ## More key configuration
 
-In our quick example, we have included the `key` as a _heredoc_
-directly into the configuration file. We can do better :)
+In our code example, we have read the key from a file with the `key_file` attribute. That is a good way, if you actually have the key in a file or if you want to mount a Kubernetes secret to a file.
```

Comment:
..., we specify a `key_file` attribute referencing a file containing the key.

Reply:
ok